labgrid/remote/exporter.py

```diff
@@ -48,6 +67,22 @@ def __attrs_post_init__(self):
         for key in self.local_params:
             del self.params[key]
         self.start_params = None
+        self._broken = None
+
+    # is something criticial failed for an export, we can mark it as
+    # permanently broken
+    @property
+    def broken(self):
+        return self._broken
+
+    @broken.setter
+    def broken(self, reason):
+        assert self._broken is None
+        assert type(reason) == str
+        assert len(reason) > 0
```

Comment:
```suggestion
        assert reason
```
Should be enough at this point.